roles/reverseproxy/tasks/server.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: install nginx
+  ansible.builtin.apt: pkg=nginx state=present
+
+- name: remove default nginx configuration
+  ansible.builtin.file:
+    path: /etc/nginx/sites-enabled/default
+    state: absent
+  notify: reload nginx
+
+- name: install the ssl key
+  ansible.builtin.copy:
+    content: "{{ reverseproxy_key }}"
+    dest: /etc/ssl/reverseproxy.key
+    mode: 0600
+
+- name: install the ssl certificate
+  ansible.builtin.copy:
```

Comment:
I think a loop works better here to bundle all cert related things together (for readability)